Configurations/release-move-tag.sh

```diff
@@ -4,23 +4,30 @@ set -e
 # Convenience script to automatically commit Package.swift after updating the checksum and move the latest tag
 latest_git_tag=$(git describe --abbrev=0) # gets the latest tag name
 commits_since_tag=$(git rev-list ${latest_git_tag}.. --count)
+
+function move_tag() {
+    long_message=$(git tag -n99 -l $latest_git_tag) # gets corresponding message
+    long_message=${long_message/$latest_git_tag} # trims tag name
+    long_message="$(echo -e "${long_message}" | sed -e 's/^[[:space:]]*//')" # trim leading whitespace
+    git add Package.swift
+    git commit -m "Update Package.swift for version ${latest_git_tag}"
+    git tag -fa $latest_git_tag -m "${long_message}"
```

Comment:
To answer your question, creating/moving the tag (git tag) in CI is probably not necessary because we don't push it. But the git add and git commit parts are needed.

Also note make release and release-move-tag.sh are not invoked in our other normal CI workflow that triggers on pull requests, which is a good thing here.. because we wouldn't want this procedure to be invoked in that case.